### ISSUE TITLE
Flag or fail on A11y issues

### ISSUE THREAD

**Issue by yschimke** [google/horologist#645]
ComposeA11yExtension collects AccessibilityState.Elements into a list.  It only looks at the properties, but doesn't do any additional checks.  We could have an optional stage to run some tests on 

- Minium touch targets are 48x48, so highlight or fail.
- Also missing content descriptions, or actions. 

Inspiration could come from https://developer.android.com/guide/topics/ui/accessibility/testing
